docs/data-hub/dataframes.md

```diff
@@ -0,0 +1,79 @@
+# Using Deep Origin DataFrames
+
+This page describes how to use Deep Origin DataFrames, which are the primary object you will use to interact with databases on Deep Origin. This page will cover:
+
+- fetching data from a Deep Origin database
+- modifying data locally
+- writing data back to Deep Origin
+
+
+!!! question "What is a Deep Origin DataFrame?"
+    A Deep Origin DataFrame is a subclass of a [pandas.DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html) that is backed by a Deep Origin database. Because it is a subclass of a pandas DataFrame, all pandas DataFrame methods work on Deep Origin DataFrames. 
```

Comment:
It would be helpful to elaborate a little more on what "backed" means.
- DataFrames can easily be created from Deep Origin databases
- DataFrames can easily be saved to Deep Origin databases
- If we stick with auto-saving, this would be very important to describe here. Actually, we'd want to style this as a danger callout. Auto save is too dangerous to keep.

Reply:
fixed